### DISCUSSION TITLE
On the Home page, if I click play on an album, it shuffles the album. How do I make it start from the beginning of the album?

### DISCUSSION THREAD

**Discussion by nothing2obvi** [jeffvli/feishin#775] [Q&A]
Basically the title. Thanks!

**Comment by iiPythonx** [comment]
Make sure that your play setting is set to Play rather then Play Shuffled in settings (which should be the default):
![image](https://github.com/user-attachments/assets/ba422285-6c7c-44db-82fe-9432cb7b7e51)

If it's already set to play and you don't have shuffle on either then it might be related to recent Navidrome sort order changes (assuming you're running Navidrome, in which case you need to update to >=0.53.2).

If you're on Jellyfin, then I'm not sure what could be causing that.

**Comment by nothing2obvi** [comment]
@iiPythonx thanks for your response. 

I'm on Navidrome `0.53.3` and my play behavior was set to `Play`. Still getting the shuffling, unfortunately.
